frontend/src/components/ModalContent.vue

```diff
@@ -1,119 +1,116 @@
+          <v-icon size="3.5rem" color="white">close</v-icon>
+        </v-btn>
+      </div>
     </v-card-title>
-    <v-card-actions>
-      <div class="buttons-list">
-        <v-row>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.licensePage"
-            target="_blank"
-          >
-            <v-icon>far fa-copyright</v-icon>
-            <p class="buttons--text">
-              Licencja {{ selectedProject.licenseName }}
-            </p>
-          </v-btn>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.githubLink"
-            target="_blank"
-          >
-            <v-icon>fab fa-github</v-icon>
-            <p class="buttons--text">Repozytorium</p>
-          </v-btn>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.websiteLink"
-            target="_blank"
+    <v-card-actions class="buttons-list unified-padding">
+      <div v-show="selectedProject.licensePage !== ''">
+        <v-btn text rounded :href="selectedProject.licensePage" target="_blank">
+          <v-icon>far fa-copyright</v-icon>
+          <span class="buttons-text"
+            >Licencja {{ selectedProject.licenseName }}</span
           >
-            <v-icon>language</v-icon>
-            <p class="buttons--text">Strona projektu</p>
-          </v-btn>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.partnerPage"
-            target="_blank"
-          >
-            <v-icon>fas fa-handshake</v-icon>
-            <p class="buttons--text">Strona partnera</p>
-          </v-btn>
-        </v-row>
+        </v-btn>
+      </div>
+      <div v-show="selectedProject.githubLink !== ''">
+        <v-btn text rounded :href="selectedProject.githubLink" target="_blank">
+          <v-icon>fab fa-github</v-icon>
+          <span class="buttons-text">Repozytorium</span>
+        </v-btn>
+      </div>
+      <div v-show="selectedProject.websiteLink !== ''">
+        <v-btn text rounded :href="selectedProject.websiteLink" target="_blank">
+          <v-icon>language</v-icon>
+          <span class="buttons-text">Strona projektu</span>
+        </v-btn>
       </div>
     </v-card-actions>
-    <div class="text-list">
-      <v-row>
-        <pre class="modal-subtitle">Partner projektu: </pre>
-        <p class="modal-subtitle">{{ selectedProject.partner }}</p>
-      </v-row>
-    </div>
-    <div class="text-list">
-      <v-row>
-        <p class="content-black">{{ selectedProject.description }}</p>
-      </v-row>
+    <v-card-text class="unified-padding">
+      <p class="text-center">{{ selectedProject.description }}</p>
+    </v-card-text>
+    <div v-show="countPartners >> 0">
```

Comment:
is `>>` it ok? not just `>` ?

Reply:
You're right. For some reason webstorm was correcting me on a singular use, but actually it seems to work fine. FYI VSC doesn't claim that kind of problem :D